server.go

```diff
@@ -13,17 +13,17 @@ import (
 
 type server struct {
 	listener                net.Listener
-	ignoreOpenSSHExtensions bool
+	powershellPath string
 }
 
-func newServer(path string, ignoreOpenSSHExtensions bool) *server {
+func newServer(powershellPath string, path string) *server {
```

Comment:
Please rename `path` to `socketPath`, and make `socketPath` the first argument (because it is the primary argument for newServer).